Source/WebCore/accessibility/AccessibilityRenderObject.cpp

```diff
@@ -718,7 +718,7 @@ String AccessibilityRenderObject::textUnderElement(AccessibilityTextUnderElement
             if (WeakPtr renderTextFragment = dynamicDowncast<RenderTextFragment>(*renderText)) {
                 // The alt attribute may be set on a text fragment through CSS, which should be honored.
                 const auto& altText = renderTextFragment->altText();
-                if (!altText.isEmpty())
+                if (!altText.isNull())
```

Comment:
This touches only `RenderTextFragment`, does the case with `content: url(...) / ""` work as expected?

I'm seeing other isEmpty() checks: https://searchfox.org/wubkat/search?q=AltText.isEmpty&path=&case=false&regexp=false

Reply:
Most of those results seem to be related to the img alt attribute rather than CSS content alt text fwiw

Reply:
I do want to make sure `content: url(...) / ""` also works properly, because I'm fairly sure that doesn't use `RenderTextFragment`

Reply:
Are there any changes to the result of this WPT test file? If not, will you add a new [interop-accessibility issue](https://github.com/web-platform-tests/interop-accessibility/issues) that we probably need to add a few more subtests?
https://wpt.fyi/results/accname/name/comp_name_from_content.html?label=master&label=experimental&aligned&q=label%3Aaccessibility